reference/catalog.v3.yml

```diff
@@ -20369,7 +20373,9 @@ definitions:
       - type: object
         properties:
           variants:
-            $ref: '#/definitions/productVariant_Put_Product'
+            type: array
```

Comment:
Can you please verify that productVariant_Put_Product should be an array? It appears to be an object.